### PR TITLE
ApplicantScreen and MilestonesScreen

### PR DESCRIPTION
**Pull request by bernardodiasc**
Required for https://github.com/x-team/auto/pull/736

### CODE REVIEW

**Comment by nicksp** [comment]
@bernardodiasc What exact help are you asking for in this PR?

**Comment by bernardodiasc** [comment]
@nicksp mostly code review and any additional change required.

File: src/components/MilestonesScreen.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  & > *:last-child {
+    padding-bottom: 65px
+  }
+`))
+
+const Block = elem.div(cmz(`
+  max-width: 920px
+  margin: 0 auto
+  padding: 65px 60px 130px
+  border-top: 1px solid #e0e0e0
```
Comment (justtry) [inline src/components/MilestonesScreen.js]: hey @bernardodiasc, will border still change color for current and past elements if we set color here?

Comment (bernardodiasc) [reply src/components/MilestonesScreen.js]: Good catch, this is required to be some of the colours variables. At the time I made this PR the colours sets PR wasn't merged yet :)

**Comment by falcoagustin** [comment]
@nicksp This is ready for re-review. I adapted it to fit with master, do you think its possible for us to merge this one?

**Comment by nicksp** [comment]
Finally jumped on reviewing that 😄

**Comment by nicksp** [comment]
Did several changes here: https://github.com/x-team/auto-ui/pull/76/commits/adbc8c5da33a272a131cb848f4047d2c7b47f5a7, https://github.com/x-team/auto-ui/pull/76/commits/076312b5a86e593183272636460cdb1c2325875a, https://github.com/x-team/auto-ui/pull/76/commits/839f2f907b41a0a7de387d0952de647def7c292c, and https://github.com/x-team/auto-ui/pull/76/commits/0b44ccd19c54fd7d5ad199921f47657487bcd819

Ready to merge now 🎉

**Comment by falcoagustin** [comment]
Awesome 🎉 😄

**Pull request merged**